LondonBritishLibrary/orient/BLorient813.xml

```diff
@@ -30,20 +31,272 @@ schematypens="http://relaxng.org/ns/structure/1.0"
+                            <textLang mainLang="gez"></textLang>
+                          </msItem>
+                          <msItem xml:id="ms_i1.43">
+                            <title type="complete" ref="LIT1011Abusak#Chapter42" xml:lang="gez">አንቀጽ፡ ፵፪።</title>
+                            <textLang mainLang="gez"></textLang>
+                          </msItem>
+                          <msItem xml:id="ms_i1.44">
+                            <title type="complete" ref="LIT1011Abusak#Chapter43" xml:lang="gez">አንቀጽ፡ ፵፫።</title>
+                            <textLang mainLang="gez"></textLang>
+                          </msItem>
+                          <msItem xml:id="ms_i1.45">
+                            <title type="complete" ref="LIT1011Abusak#Chapter44" xml:lang="gez">አንቀጽ፡ ፵፬።</title>
+                            <textLang mainLang="gez"></textLang>
+                          </msItem>
+                          <msItem xml:id="ms_i1.46">
+                            <title type="complete" ref="LIT1011Abusak#Chapter45" xml:lang="gez">አንቀጽ፡ ፵፭።</title>
+                            <textLang mainLang="gez"></textLang>
+                          </msItem>
+                          <msItem xml:id="ms_i1.47">
+                            <title type="complete" ref="LIT1011Abusak#Chapter46" xml:lang="gez">አንቀጽ፡ ፵፮።</title>
+                            <textLang mainLang="gez"></textLang>
+                            <note>Chapter 47 is missing.</note>
+                          </msItem>
+                          <msItem xml:id="ms_i1.48">
+                            <locus from="89v"></locus>
+                            <title type="complete" ref="LIT1011Abusak#Chapter48" xml:lang="gez">አንቀጽ፡ ፵ወ</title>
+                            <textLang mainLang="gez"></textLang>
+                            <note>Chapter 49 is missing.</note>
+                          </msItem>
+                          <msItem xml:id="ms_i1.49">
+                            <locus from="148r"></locus>
+                            <title type="complete" ref="LIT1011Abusak#Chapter50" xml:lang="gez">አንቀጽ፡ ፶።</title>
+                            <textLang mainLang="gez"></textLang>
+                            <note>Chapter 51 is missing.</note>
+
+                            <msItem xml:id="ms_i1.49.1">
+                              <title type="complete" xml:lang="en">The list of the patriarchs
+                              of Alexandria</title>
+                              <textLang mainLang="gez"></textLang>
+                              <note>The list ends with <persName ref="PRS6752Mark"></persName>.</note>
+                            </msItem>
+                            <msItem xml:id="ms_i1.49.2">
+                              <locus from="173v"></locus>
+                              <title type="complete" xml:lang="en">The Seven Wonders of the World</title>
```

Comment:
I see that this is also attested in another BL ʾAbušākər manuscript. Could you find some information on this textual unit? Is it normally attested in ʾAbušākər manuscripts and could be included as a subunit in the work record?

Reply:
Yes, you are right, Wonders and list of Patriarchs seems to belong to the normal structure. I wull add them to the work record.